skyportal/tests/rate_limiting/test_rate_limiting.py

```diff
@@ -1,26 +1,58 @@
 import time
+import socket
+import requests
+from baselayer.app.env import load_env
 from skyportal.tests import api
 
 
+_, cfg = load_env()
+
+s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+s.connect(("8.8.8.8", 53))
+localhost_external_ip = s.getsockname()[0]
+s.close()
+
+
 def test_api_rate_limiting(view_only_token):
+    # In case this test gets run after those defined below
+    time.sleep(5)
```

Comment:
Instead of an arbitrary sleep, perhaps there should be a loop *at the end* of this test that hits the API until it becomes available again?  That way, we are not reliant on test order, since the test becomes self-encapsulated.